specs/deneb/polynomial-commitments.md

```diff
@@ -104,7 +104,6 @@ but reusing the `mainnet` settings in public networks is a critical security req
 | Name | Value |
 | - | - |
 | `KZG_SETUP_G2_LENGTH` | `65` |
-| `KZG_SETUP_G1` | `Vector[G1Point, FIELD_ELEMENTS_PER_BLOB]`, contents TBD |
 | `KZG_SETUP_G2` | `Vector[G2Point, KZG_SETUP_G2_LENGTH]`, contents TBD |
 | `KZG_SETUP_LAGRANGE` | `Vector[G1Point, FIELD_ELEMENTS_PER_BLOB]`, contents TBD |
```

Comment:
I think we should rename these to `KZG_SETUP_G1_LAGRANGE` and `KZG_SETUP_G2_MONOMIAL`.

Reply:
I agree, this clarity is helpful to understand what is happening under the hood.